assets/css/admin-custom.css

```diff
@@ -117,3 +117,54 @@ div.question_boolean_fields { margin-bottom: 10px; }
 .sensei-dismissible-link {
     text-decoration: none;
 }
+
+/**
+ * Sensei custom navigation
+ */
+.sensei-custom-navigation {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: flex-start;
+	color: #000000;
+	margin-bottom: 20px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__title h1 {
+	color: #1E1E1E;
+	font-size: 20px;
+	margin-right: 25px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: center;
+	justify-content: flex-start;
```

Comment:
Maybe there should be a bit more space between the nav elements when the navigation drops to multiple rows? The bottom row is too close to the top one, also the navigation is too close to the title. One possible solution would be to use `gap: 20px;` and tweak the margins accordingly.

Before:
<img width="496" alt="Screenshot on 2022-01-24 at 16-22-56" src="https://user-images.githubusercontent.com/1612178/150800830-dc9f1be0-ae3f-4693-92d4-8c01e0a46368.png">

After:
<img width="499" alt="Screenshot on 2022-01-24 at 16-22-26" src="https://user-images.githubusercontent.com/1612178/150800927-d96d5095-841d-4c21-87f6-1ee766784963.png">

Reply:
Fixed with the layout change: https://github.com/Automattic/sensei/pull/4672/commits/7e32384a664b0e7a92da10bdf7d45b98d128f026